libs/menu/src/TopNavigation.tsx

```diff
@@ -438,7 +438,7 @@ export function TopNavigationItem({
           <div className="flex flex-col flex-wrap items-center">
             <Link {...props} to={to || "#"} className={cn + ` items-center`} onClick={() => setOpened(!opened)}>
               {title}
-              {opened ? expanderCloseIcon : expanderOpenIcon}
+              {opened ? expanderOpenIcon : expanderCloseIcon}
```

Comment:
This seems like the old condition was ok, but icons were reversed. `expanderOpen` should be the one that is shown when user needs to open the dropdown/card/collapse, not when it's already opened